lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -104,9 +122,5 @@ defmodule Phoenix.Endpoint.CowboyHandler do
     end
   end
 
-  defp info(scheme, endpoint, config) do
```

Comment:
That... is unhappily surprising. Is there a good spot to drop tests in to showcase this behavior while I'm reverting the logic back?